dataplane/standalone/translator.cc

```diff
@@ -127,6 +135,9 @@ sai_status_t Translator::set_attribute(sai_object_type_t type,
   } else {
     status = this->apis[idStr]->set_attribute(attr);
   }
+  if (status == SAI_STATUS_SUCCESS) {
+    this->attrMgr->set_attribute(idStr, attr);
+  }
```

Comment:
What's the difference between this attrMgr and the one in `apis`? It looks like they're the same?

Reply:
attrMgr only stores values in a map. apis actually does the work related setting the attribute